pallets/liquidity-pools-gateway/src/message_processing.rs

```diff
@@ -47,8 +49,8 @@ pub struct InboundProcessingInfo<T: Config> {
 impl<T: Config> Pallet<T> {
 	/// Calculates and returns the proof count required for processing one
 	/// inbound message.
-	fn get_expected_proof_count(domain: &Domain) -> Result<u32, DispatchError> {
-		let routers = Routers::<T>::get(domain).ok_or(Error::<T>::RoutersNotFound)?;
+	fn get_expected_proof_count() -> Result<u32, DispatchError> {
+		let routers = Routers::<T>::get();
```

Comment:
And here, for example, I should instead use `T::RouterId::for_domain...`?

Reply:
Yes, because we want to get only the valid routers for the domain where the message comes